creator/refferalTokens/models.py

```diff
@@ -0,0 +1,49 @@
+import uuid
+from django.db import models
+from creator.studies.models import Study
+
+User = get_user_model()
+
+
+class ReferralToken(models.Model):
+    """
+    An Event that involved some user, study, file, version, or a combination
+    of them.
+    """
+
+    class Meta:
+        permissions = [
+            ("add_refferaltoken", "Can generate a new referral token",)
```

Comment:
This permission should be generated by [default](https://docs.djangoproject.com/en/3.0/topics/auth/default/#default-permissions).